css/style.css

```diff
@@ -128,14 +132,14 @@ header nav ul li a {
 .blue {
   background: #033048;
   color: #fff;
-  font-size: 24px;
+  font-size: 0.923rem;
 }
 #benefits {
   padding: 100px 0 80px;
```

Comment:
I'd recommend using `min-height` for your `#hero` selector rather than hardcoding the `700px` `height`